app/src/main/java/com/github/sdpcoachme/schedule/CreateEventActivity.kt

```diff
@@ -223,11 +231,23 @@ fun NewEvent(store: CachingStore) {
                                 val toast = Toast.makeText(context, "Start date must be before end date", Toast.LENGTH_SHORT)
                                 toast.show()
                             } else {
-                                EventOps.addEvent(event, store).thenAccept {
+                                EventOps.addEvent(event, store)/*.thenCompose {
+                                    val organiser = database.getCurrentEmail().replace('.', ',')
```

Comment:
Will this commented out code snipped be needed in a future pr? Otherwise I would recommend to delete it.

Reply:
Yes, this will be used for the implementation of the next PR.